addOns/llm/src/main/javahelp/org/zaproxy/addon/llm/resources/help/contents/about.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Simple Example - About
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Simple Example - About</H1>
+
+<H2>Source Code</H2>
+<a href="https://github.com/zaproxy/zap-extensions/tree/main/addOns/simpleexample">https://github.com/zaproxy/zap-extensions/tree/main/addOns/simpleexample</a>
+
+<H2>Authors</H2>
+ZAP Dev Team
+
+<H2>History</H2>
+
+<H3>Version 2</H3>
+Updated to include example 'about' page.
+
+<H3>Version 1</H3>
+First Version
```

Comment:
All the help components need updating.

All the help_* folders and contents should be removed from the PR

Reply:
Same for messages.properties. Only the base file should be included.